assets/src/types.ts

```diff
@@ -32,11 +32,14 @@ export type Customer = {
   first_seen?: any;
   host?: string;
   ip?: string;
+  // Date
   last_seen: string;
+  // Datetime
+  last_seen_at?: string;
```

Comment:
Just curious - is there a reason why we have two `last_seen` properties? From what I can tell, they're different formats (date/datetime), but is there a reason why we don't just always have `last_seen_at` and make sure that it's not optional? I imagine that it would be easier if there was only one property.

Reply:
yeah i'd like to deprecate the `last_seen` date property, it was something we set up really early on and haven't had a chance to remove it in favor of `last_seen_at`